webapi/src/main/scala/org/knora/webapi/messages/util/rdf/ImprovedRdfModel.scala

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright © 2021 - 2024 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.messages.util.rdf
```

Comment:
IMHO we should move the file somewhere into the `org.knora.webapi.slice.common.repo` package.

Reason being is that the new `Repository` implementation should make use of that as your already did with the `KnoraProject`.

Reply:
absolutely!